src/cards/ha-weather-card.js

```diff
@@ -128,7 +129,10 @@ class HaWeatherCard extends
             <div class="forecast">
               <template is="dom-repeat" items="[[forecast]]">
                 <div>
-                  <div class="weekday">[[computeDateTime(item.datetime)]]</div>
+                  <div class="weekday">[[computeDate(item.datetime)]]</div>
```

Comment:
dom-if?

Reply:
No. The day of the week is always displayed. The hour is displayed in addition to the day of the week for hourly forecasts.
An example hourly:
![hourly](https://user-images.githubusercontent.com/33804747/41288974-03eb5378-6e6a-11e8-908b-2eaee41cafac.jpg)
An example daily:
![daily](https://user-images.githubusercontent.com/33804747/41289062-48ea8138-6e6a-11e8-8b91-de8988308054.jpg)

Reply:
so simply add a <br> and move it to the existing class?

also add a screenshot of it to PR description